.changeset/frowny-paths-hear.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/datastore": minor
+---
+---
+"section": legacy
+---
```

Comment:
We're in a transition period for changesets, this is the new format, with Fluid-specific settings in the same front matter "container", prefixed with double underscore.
```suggestion
---
"@fluidframework/datastore": minor
"__section": legacy
---
```